src/User/LoginForm.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { useHistory, Link } from "react-router-dom";
+import Input from "../Input";
+import { loginUser } from "../api";
+import {
+  StyledCardHeading,
+  StyledForm,
+  StyledCard
+} from "../StyledFormComponents";
+
+export const StyledLink = styled(Link)`
+  color: #7a517d;
+  padding: 10px;
+`;
+
+const LoginForm = ({ setCookie }) => {
+  let history = useHistory();
+  const [user, setUser] = useState({
+    username: "",
+    password: ""
+  });
+
+  const handleChange = event => {
+    const { name, value } = event.target;
+    setUser({ ...user, [name]: value });
+  };
+
+  const submitForm = async () => {
+    try {
+      const token = await loginUser(user);
+      setCookie("sessionToken", token, { path: "/" });
```

Comment:
I would abstract this out a level, so that this form does not need to know what the cookie information is, only the token.

eg pass from parent:
`setToken = {token => setCookie("sessionToken", token, { path: "/" });}`

here:
`setToken(token)`

Reply:
This means that the component is less tied to the implementation (cookies) and the constants such as the name of the token can be kept in the same file, improving maintainability

Reply:
ok, done! :)